pom.xml

```diff
@@ -553,11 +553,17 @@
         <preview.version.jakarta.servlet.jsp.jakarta-servlet-jsp-api>3.0.0</preview.version.jakarta.servlet.jsp.jakarta-servlet-jsp-api>
         <preview.version.jakarta.servlet.jsp.jstl.jakarta-servlet-jsp-jstl-api>3.0.0</preview.version.jakarta.servlet.jsp.jstl.jakarta-servlet-jsp-jstl-api>
         <preview.version.jakarta.transaction.jakarta-transaction-api>2.0.0</preview.version.jakarta.transaction.jakarta-transaction-api>
+        <preview.version.jakarta.xml.bind-api>3.0.1</preview.version.jakarta.xml.bind-api>
         <preview.version.jakarta.validation.jakarta-validation-api>3.0.0</preview.version.jakarta.validation.jakarta-validation-api>
         <preview.version.jakarta.websocket.jakarta-websocket-api>2.0.0</preview.version.jakarta.websocket.jakarta-websocket-api>
         <preview.version.jakarta.ws.rs.jakarta-ws-rs-api>3.1.0</preview.version.jakarta.ws.rs.jakarta-ws-rs-api>
         <!--<preview.version.jakarta.xml.bind.jakarta-xml-bind-api>3.0.0</preview.version.jakarta.xml.bind.jakarta-xml-bind-api>-->
+        <preview.version.net.shibboleth.utilities.java-support>8.0.0</preview.version.net.shibboleth.utilities.java-support>
+        <preview.version.org.apache.cxf>3.5.2-jbossorg</preview.version.org.apache.cxf>
```

Comment:
There is a missing "-1" here. It should be `3.5.2-jbossorg-1`.

Reply:
Fixed.